### PR TITLE
The activesupport gem is a runtime dependency

### PR DESCRIPTION
**Pull request by billhorsman**
not just a development dependency, because of the use of String#blank?
and Array.wrap in MailInterceptor::Interceptor.

### CODE REVIEW

File: lib/mail_interceptor.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require "active_support/all"
```
Comment (vipulnsward) [inline lib/mail_interceptor.rb]: Only following requires should suffice - 
```ruby
require 'active_support/core_ext/object/blank'
require 'active_support/core_ext/array'
```
Can you update that?

File: mail_interceptor.gemspec [hunk 2]
```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "minitest"
-  spec.add_development_dependency "activesupport"
+  spec.add_runtime_dependency "activesupport"
```
Comment (vipulnsward) [inline mail_interceptor.gemspec]: can be moved to the top/bottom of dependencies, so dependencies are clubbed.

**Comment by neerajdotname** [comment]
Looks good to me. 

@vipulnsward please take a look.

**Comment by vipulnsward** [comment]
Thanks @billhorsman !

**Pull request merged**